code/game/machinery/iv_drip.dm

```diff
@@ -347,6 +341,7 @@
 /obj/machinery/iv_drip/plumbing/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/plumbing/iv_drip, anchored)
+	AddComponent(/datum/component/simple_rotation)
 
```

Comment:
```suggestion
/obj/machinery/iv_drip/plumbing/AltClick(mob/user)
	return ..() // This hotkey is BLACKLISTED since it's used by /datum/component/simple_rotation
```

Add this back in then since we want any future coders to see that AltClick is being used by the component.  Since we are doing `return ..()` it should work fine.